pyproject.toml

```diff
@@ -26,7 +26,7 @@ environment = {CFLAGS = "-O3 -g1 -pipe -fPIC -flto -march=armv8-a -mtune=cortex-
 
 [tool.cibuildwheel.windows]
 archs = ["AMD64", "x86"]
-test-command = "python -c \"import lupa\" && python -c \"import lupa.lua54\" && python -c \"import lupa.luajit21\" "
+test-command = "python -c 'import lupa' && python -c 'import lupa.lua54' && python -c 'import lupa.luajit21' "
```

Comment:
there is another (same) `test-command` in common section (line 9).

Reply:
Turns out that this change doesn't work. It still splits the comment at space boundaries, ignoring the single quotes. I'll revert it.